### DISCUSSION TITLE
Migrated object store to S3: What can I delete on the local instance?

### DISCUSSION THREAD

**Discussion by ralfr** [mastodon/mastodon#21523] [Q&A]
Question to those of you who've migrated object store to cloud providers (S3, etc.): 

Is it safe to delete everything in `live/public/system`? It seems to me that once you've got S3 configured, Mastodon fetches everything in the `/system` path from the remote storage location.

**Comment by marians** [comment]
@ralfr Have you ever found an answer to your question? Am I right to assume that the local files will not be touched any more after a migration to object storage?

**Comment by dlukt** [comment]
How *do* you even migrate?
I moved the live/public/system files to minio storage
I guess I need to reconfigure mastodon now?

**Reply by dlukt** [reply]
And to answer this question.
```bash
cd live/public/system
mc migrate ./ s3/<bucket-name>
```
Change .env.production
```
S3_ENABLED=true
S3_ENDPOINT=https://s3.domain.tld         # nginx terminates TLS
# Leave path-style (default). Do NOT set S3_OVERRIDE_PATH_STYLE for this.
S3_BUCKET=<bucket-name>
S3_PROTOCOL=https
S3_ALIAS_HOST=s3.domain.tld/<bucket-name>

AWS_ACCESS_KEY_ID=<key>
AWS_SECRET_ACCESS_KEY=<secret>
```

**Comment by dlukt** [comment]
To answer the question: Yes
If you migrated everything to s3 storage it's safe to delete the files
